action.yml

```
@@ -115,8 +115,10 @@ runs:
         echo "::add-mask::$(cat branch_out | jq -r '.connection_uris[0].connection_parameters.password')"
         echo "branch create out:"
         cat branch_out
-        echo "branch create err:"
-        cat branch_err
+        if [[ $(cat branch_err) ]]; then
```

let's do the actual file exists check if it's the case
```suggestion
        if [[ -f branch_err ]]; then
```